projects/observability/src/pages/apis/backend-detail/traces/backend-trace-list.dashboard.ts

```diff
@@ -29,6 +29,16 @@ export const backendTraceListDashboard = {
           attribute: 'name'
         }
       },
+      {
+        type: 'table-widget-column',
+        title: 'Operation',
+        width: '1',
+        filterable: true,
+        value: {
+          type: 'attribute-specification',
+          attribute: 'backend_operation'
```

Comment:
not sure if this has been merged on the backend side yet, but our naming convention for attribute keys is camelCase, not snake_case

Reply:
No, it has not been merged in backend yet. Thanks, I will use camelCase for setting this attribute in config-bootstrapper.